### PR TITLE
React styleguidist setup

### PR DESCRIPTION
**Pull request by naj21**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# Installation
+# Idea Directory
+
+This repositroy contains the idea-directory web project. A project that provides a platform where tech related ideas can be dispensed by users.
```
Comment (hackthievist) [inline README.md:3]:
```suggestion
This repository contains the idea-directory web project. A project that provides a platform where tech related ideas can be dispensed by users.
```

**Comment by hackthievist** [comment]
Are you setting up an entire PR just to add a package?

**Comment by hackthievist** [comment]
Also, can you subsequently briefly describe what changes your PR aims to make and the motivation behind the changes, it gives some context and makes reviewing easierrrrr

**Comment by naj21** [comment]
> Are you setting up an entire PR just to add a package?

Yes, I have a "valid" excuse

**Comment by naj21** [comment]
> Also, can you subsequently briefly describe what changes your PR aims to make and the motivation behind the changes, it gives some context and makes reviewing easierrrrr

**Comment by hackthievist** [comment]
> > Are you setting up an entire PR just to add a package?
> 
> Yes, I have a "valid" excuse
> 
> Na I don't, just realized my excuse was in the last PR.

smh

**Comment by naj21** [comment]
> > > Are you setting up an entire PR just to add a package?
> > 
> > 
> > Yes, I have a "valid" excuse
> > Na I don't, just realized my excuse was in the last PR.
> 
> smh

Are you meant to do that here, this is a professional space!

**Comment by hackthievist** [comment]
> Are you meant to do that here, this is a professional space!

whatever

**Comment by naj21** [comment]
> > Are you meant to do that here, this is a professional space!
> 
> whatever

I can remove you as a reviewer and collaborator. Who's the boss now?!

**Pull request closed without merging**